src/main/java/uk/gov/hmcts/reform/civil/service/notification/defendantresponse/fulldefence/FullDefenceApplicantSolicitorOneCCUnspecNotifier.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.civil.service.notification.defendantresponse.fulldefence;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+
+@Component
+public class FullDefenceApplicantSolicitorOneCCUnspecNotifier extends FullDefenceSolicitorUnspecNotifier {
+
+    //NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CC
```

Comment:
is this to let people know of the event? if so shouldnt the class name reflect that?
also spacing below

Reply:
LGTM OTHERWISE